### DISCUSSION TITLE
🔗 The Gap (between widgets) parameter in Site Settings should be responsive (Duplicate of #25155)

### DISCUSSION THREAD

**Discussion by VadymPogorelov** [elementor/elementor#28338] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

The Gap parameter set in Site Settings for GRID layouts (e.g. 30 pixels) is fine for desktops, tolerable on tablets (although you already want to reduce it) and proves to be too large on mobile devices https://prnt.sc/Ll26qkEFdmmZ

### Describe the solution you'd like.

It would be a good solution to make this parameter responsive.

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by nicholaszein** [comment]
Hello there, @VadymPogorelov!

Thank you for your submission! 🙏 

### 🔗 🚀 There is already a similar request submitted by someone here #25155

Feel free to continue the conversation and cast your **⬆️ Upvote** there! 👉

### 🔍 Please, in the future, remember to check if there isn't any thread similar to yours already opened. 

Kind regards
